patches/createsend-node+0.8.2.patch

```diff
@@ -0,0 +1,39 @@
+diff --git a/node_modules/createsend-node/lib/interface/utils.js b/node_modules/createsend-node/lib/interface/utils.js
+index acc6561..b39f481 100644
+--- a/node_modules/createsend-node/lib/interface/utils.js
++++ b/node_modules/createsend-node/lib/interface/utils.js
+@@ -44,22 +44,20 @@ module.exports = {
+                 if (error) { return callback(error); }
+
+                 var code = response.statusCode;
+-                if (code == 200 || code == 201 || code == 202) {
+-                    if (body) {
+-                        try {
+-                            callback(null, JSON.parse(body));
+-                        } catch (e) {
+-                            callback(new Error('Unable to parse JSON.  Error: ' + e + '\nBody:\n' + body));
+-                        }
+-                    } else {
+-                        callback(null);
++                if (body) {
++                    try {
++                        body = JSON.parse(body);
++                        body.headers = response.headers;
```

Comment:
This is the crux of my patch.  I parse the body and then add the headers to it.  In case it's easier to just look at the resulting code, it's below.  I consolidated the body parsing to make it cleaner.
```
        request(
            options,
            function (error, response, body) {
                if (error) { return callback(error); }

                var code = response.statusCode;
                if (body) {
                    try {
                        body = JSON.parse(body);
                        body.headers = response.headers;
                    } catch (e) {
                        callback(new Error('Unable to parse JSON.  Error: ' + e + '\nBody:\n' + body));
                        return;
                    }
                }

                if (code == 200 || code == 201 || code == 202) {
                    callback(null, body);
                } else {
                    callback(body);
                }
            }
        );
```